lastadapter/src/main/java/com/github/nitrico/lastadapter/LastAdapter.kt

```diff
@@ -35,7 +35,7 @@ class LastAdapter(private val list: List<Any>,
     private val DATA_INVALIDATION = Any()
     private val callback = ObservableListCallback(this)
     private var recyclerView: RecyclerView? = null
-    private var inflater: LayoutInflater? = null
+    private lateinit var inflater: LayoutInflater
```

Comment:
I made it `lateinit` as I felt the real nature of this is late initialization, and won't be nullable ever.